AvocadoToast/Sources/UI/OrderInput/OrderForm.swift

```diff
@@ -81,25 +77,31 @@ struct OrderForm: View {
     }
 
     func submitOrder() {
-        history.add(CompleteOrder(order: order))
+        processor.process(order)
         withAnimation {
-            order = Order()
+            order = draftOrder
         }
     }
 }
 
-struct OrderForm_Previews: PreviewProvider {
-    static var previews: some View {
-        Group {
-            OrderForm(History())
-            OrderForm(History())
-                .environment(\.sizeCategory, .extraExtraLarge)
-                .environment(\.colorScheme, .dark)
-                .environment(\.locale, Locale(identifier: "en_US"))
-            OrderForm(History())
-                .environment(\.sizeCategory, .small)
-                .environment(\.colorScheme, .light)
-                .environment(\.locale, Locale(identifier: "he_IL"))
-        }
+extension OrderForm {
+    private var draftOrder: Order {
```

Comment:
This does't add much value. We can remove it and use `order = .default` inside `func submitOrder()`